_data/accessibility-tests/search.yml

```
@@ -0,0 +1,175 @@
+title: Search
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey the search button's meaning.
+    summary_additional: When using the search button, "search" text or a magnifying glass icon convey the button's meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Search button background and text meet color contrast requirements. 
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search button text, the contrast between background and lettering is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: Search button background and icon meet color contrast requirements.
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search icon button, the contrast between the background and icon is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.11
+  - summary: The height and width of the search button touch target is at least 44 pixels.
+    summary_additional: |
+      Touch target size can be calculated by adding the height or width of the element and including the spacing units that separate it from adjacent targets.
+      When you select "inspect element" to review the button's size, you can see the the button and it's spacing is at least 44 pixels wide by 44 pixels tall.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+  - summary: Search component matches device orientation
+    summary_additional: |
+      On mobile devices and tablets, the search component should work in both portrait and landscape orientation with no loss of functionality.
+      When you rotate between portrait mode and landscape mode on a mobile device, the search button and field are not restricted to a single view.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.4
+  - summary: Page context does not change before search submission.
+    summary_additional: |
+      Unless notified beforehand, page content stays the same until you submit the search. If search results are opened within a new window, an advanced warning should be given alongside search instructions, in the aria-label, or button text.
+      instructions in the aria-label or button text.
```

Looks like some bonus text was included accidentally. Also, wondering if we can remove a couple commas (this one is not a blocker).
```suggestion
      Unless notified beforehand, page content stays the same until you submit the search. If search results are opened within a new window, an advanced warning should be given alongside search instructions in the `aria-label` or button text.
```

> [!Note]
> This is a continuation of this [comment thread](https://github.com/uswds/uswds-site/pull/2694#discussion_r1630271939).